modules/core/src/main/kotlin/datamaintain/core/step/executor/Executor.kt

```diff
@@ -29,10 +30,9 @@ class Executor(private val context: Context) {
                     context.reportBuilder.addExecutedScript(executedScript)
 
                 if (executedScript.executionStatus == ExecutionStatus.KO) {
-                    context.reportBuilder.inError(executedScript)
-                    logger.info { "${executedScript.name} has not been correctly executed" }
+                    logger.info { "" }
```

Comment:
Logging nothing does not seem very useful. Since the message has been transferred to ```DatamaintainScriptExecutionException```, this line should be deleted.

Reply:
If the Execute step work correctly we print an empty line for separate executor from other step logs :
```
logger.info { "" }
return context.reportBuilder.toReport()
```

I just do the same in case of error, otherwise there is no line separation between executor log and Datamainain Summary :
```
Executes scripts..
01_file.js executed

Error at step EXECUTE
Summary => 
```

Reply:
Oh ok! Let's merge then